src/backend/tests/it/utils/assertion.rs

```diff
@@ -1,5 +1,6 @@
 use shared::types::custom_token::CustomToken;
 use shared::types::token::UserToken;
+use shared::types::TokenTimestamp;
 
 pub fn assert_tokens_eq(results_tokens: Vec<UserToken>, expected_tokens: Vec<UserToken>) {
```

Comment:
```suggestion
pub fn assert_tokens_eq(results_tokens: &[UserToken], expected_tokens: &[UserToken]) {
```
If you pass by reference there is no need to clone.  Also, you have more flexibility as you can use any slice, not just a vector.